solver/src/solver/http_solver/settlement.rs

```diff
@@ -235,29 +232,27 @@ mod tests {
             fee: 5.into(),
             settlement_handling: amm_handler.clone(),
         };
-        let constant_product_orders = hashmap! { "amm0".to_string() => constant_product_order };
+        let constant_product_orders = hashmap! { 0 => constant_product_order };
 
         let executed_order = ExecutedOrderModel {
             exec_buy_amount: 6.into(),
             exec_sell_amount: 7.into(),
         };
-        let updated_uniswap = UpdatedUniswapModel {
-            balance_update1: 8,
-            balance_update2: -9,
+        let updated_uniswap = UpdatedAmmModel {
+            updates: hashmap! { t0 => 8, t1 => -9 },
             exec_plan: Some(ExecutionPlanCoordinatesModel {
                 sequence: 0,
                 position: 0,
             }),
         };
         let settled = SettledBatchAuctionModel {
-            orders: hashmap! { "lo0".to_string() => executed_order },
-            uniswaps: hashmap! { "amm0".to_string() => updated_uniswap },
-            ref_token: "t0".to_string(),
-            prices: hashmap! { "t0".to_string() => Price(10.0), "t1".to_string() => Price(11.0) },
+            orders: hashmap! { 0 => executed_order },
+            amms: hashmap! { 0 => updated_uniswap },
```

Comment:
Note to self: Going to have to double check/ be careful when building this map that we don't overwrite the CPOs with the WPOs since we are using indices.